docker/scripts/start_template_renderer.sh

```diff
@@ -12,4 +12,5 @@ then
     sudo -E -H -u musicbrainz carton exec -- ./script/compile_resources.sh server
 fi
 
-exec sudo -E -H -u musicbrainz carton exec -- ./script/start_renderer.pl
+exec sudo -E -H -u musicbrainz \
+    carton exec -- ./script/start_renderer.pl 2>&1
```

Comment:
In commit message you wrote:
> (start_template_renderer.sh is also used by the production images, but we don't
use svlogd there.)

The standard error stream of the website images is captured separately in Docker logs.

This redirection should probably be made in `docker/musicbrainz-tests/template-renderer.service` instead.

Reply:
Right, I'll move it!